src/Microsoft.DotNet.Interactive.Data/TabularDataExtensions.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Linq;
+using Microsoft.Data.Analysis;
+
+// ReSharper disable CheckNamespace
+namespace Microsoft.DotNet.Interactive.Formatting.TabularData;
+
+public static class TabularDataExtensions
+{
+    public static DataFrame ToDataFrame(this TabularDataResource tabularDataResource)
```

Comment:
Let's discuss before we add a new package just for a few extension methods.

Reply:
The package is added as that will contain also some additional formatters for it and will be consumed by kql and sql packages. This could be part of the main Dotnet.Interactive package

Reply:
If we're considering removing `TabularDataResource`, then adding a package to create a method that lets us work with both `DataFrame` and `TabularDataResource` together is just extra work that will have to be undone later.

Also, I only see this being used in tests, which tells me it can be a utility method in the test code for now.